tests/virt-handler_test.go

```diff
@@ -94,7 +92,7 @@ var _ = Describe("[sig-compute]virt-handler", decorators.SigCompute, func() {
 		}
 
 		By("Running the VMI")
-		vmi := libvmifact.NewFedora(libnet.WithMasqueradeNetworking())
+		vmi := libvmifact.NewFedora(libnet.WithMasqueradeNetworking(), libvmi.WithAutoattachGraphicsDevice(true))
```

Comment:
Guest agent fails without graphics devices?

Reply:
No, but part of the test is to ensure that VNC works.